src/Workspaces/Core/Portable/Shared/Extensions/DocumentExtensions.cs

```diff
@@ -59,6 +59,10 @@ public static bool IsOpen(this Document document)
                 }
 
                 var node = token.Parent.AncestorsAndSelf().FirstOrDefault(a => a.FullSpan.Contains(span));
+                if (node == null)
```

Comment:
Hrmmm... how could node be null?  When would we have a span that wasn't contained in the full span of some node in the document?